python/pyspark/sql/pandas/serializers.py

```diff
@@ -1200,6 +1200,72 @@ def __repr__(self):
         return "ArrowStreamAggArrowUDFSerializer"
 
 
+# Serializer for SQL_GROUPED_AGG_ARROW_ITER_UDF
+class ArrowStreamAggArrowIterUDFSerializer(ArrowStreamArrowUDFSerializer):
```

Comment:
we should consolidate it with `ArrowStreamAggArrowUDFSerializer`: make `ArrowStreamAggArrowUDFSerializer` output the iterator and adjust the wrapper of `SQL_GROUPED_AGG_ARROW_UDF` and `SQL_WINDOW_AGG_ARROW_UDF`

Reply:
we can, please create a ticket for it